<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialPortDialog</class>
 <widget class="QDialog" name="serialPortDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="criGroupBox">
     <property name="title">
      <string>CRI/CRIN</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Serial Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="criSerialPortBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Baud Rate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="criBaudBox">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>Baud1200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud115200</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Parity</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="criParityBox">
        <item>
         <property name="text">
          <string>NoParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EvenParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OddParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SpaceParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MarkParity</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="criStopBox">
        <item>
         <property name="text">
          <string>OneStop</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OneAndHalfStop</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TwoStop</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Flow Control</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="criFlowBox">
        <item>
         <property name="text">
          <string>NoFlowControl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HardwareControl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SoftwareControl</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Data Bits</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="criDataBitsBox">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>Data5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data8</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>CANCEL</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="moreButton">
       <property name="text">
        <string>More</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="pumpGroupBox">
     <property name="title">
      <string>Pump</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Serial Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="pumpSerialPortBox">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Baud Rate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="pumpBaudBox">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>Baud1200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud115200</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Parity</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="pumpParityBox">
        <item>
         <property name="text">
          <string>NoParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EvenParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OddParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SpaceParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MarkParity</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="pumpStopBox">
        <item>
         <property name="text">
          <string>OneStop</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OneAndHalfStop</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TwoStop</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Flow Control</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="pumpFlowBox">
        <item>
         <property name="text">
          <string>NoFlowControl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HardwareControl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SoftwareControl</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Data Bits</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="pumpDataBitsBox">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>Data5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data8</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="piezoGroupBox">
     <property name="title">
      <string>Piezo</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Serial Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="piezoSerialPortBox">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Baud Rate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="piezoBaudBox">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>Baud1200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud115200</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Parity</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="piezoParityBox">
        <item>
         <property name="text">
          <string>NoParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EvenParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OddParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SpaceParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MarkParity</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="piezoStopBox">
        <item>
         <property name="text">
          <string>OneStop</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OneAndHalfStop</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TwoStop</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Flow Control</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="piezoFlowBox">
        <item>
         <property name="text">
          <string>NoFlowControl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HardwareControl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SoftwareControl</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Data Bits</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="piezoDataBitsBox">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>Data5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data8</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="otherGroupBox">
     <property name="title">
      <string>Other</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Serial Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="otherSerialPortBox">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Baud Rate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="otherBaudBox">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>Baud1200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baud115200</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Parity</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="otherParityBox">
        <item>
         <property name="text">
          <string>NoParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EvenParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OddParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SpaceParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MarkParity</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="otherStopBox">
        <item>
         <property name="text">
          <string>OneStop</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OneAndHalfStop</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TwoStop</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Flow Control</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="otherFlowBox">
        <item>
         <property name="text">
          <string>NoFlowControl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HardwareControl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SoftwareControl</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Data Bits</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="otherDataBitsBox">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>Data5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data8</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>moreButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>otherGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>644</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>serialPortDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moreButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>pumpGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moreButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>piezoGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
